### ISSUE TITLE
Continuation of Docker Hub hosting

### ISSUE THREAD

**Issue by chrisgervang** [halverneus/static-file-server#93]
Hey there, I was wondering if this docker image will continue to be hosted on [docker hub](https://hub.docker.com/r/halverneus/static-file-server) in the coming months? It's early days in understanding the implications discussed in [this article](https://blog.alexellis.io/docker-is-deleting-open-source-images/). I don't expect this project to know more than anyone else, but I'd like to discuss possible mitigations / alternatives for hosting just in case.

**Comment by halverneus** [comment]
Fair point and thank you for bringing it to my attention. In the next few weeks I'll look at updating the build chain to include additional hosting sites and include their links in the README.md file. I'll keep this issue open, in the meantime. Since September, I've taken on more work than I'm capable of delivering, but I should have free time in the coming weeks (finally) to do some overdue updates and releases to this project.

It doesn't sound like this is going to be a large issue, if I'm interpreting the intent correctly. This is a personal project that I've used at all my jobs since originally developing it (so it isn't part of an organization and I have no path or intent towards monetizing). Still, I agree that this project should have some fallback image hosting options, just in case (plus, it might help reach a larger "audience" and provide more people value).
